core/templates/components/question-directives/question-editor/question-editor.component.ts

```diff
@@ -166,7 +168,6 @@ angular.module('oppia').component('questionEditor', {
       };
 
       ctrl.saveHints = function(displayedValue) {
-        $rootScope.$applyAsync();
         _updateQuestion(function() {
           StateEditorService.setInteractionHints(
             angular.copy(displayedValue));
```

Comment:
Hm, I actually meant to move it after this line.

Reply:
okay